src/handbook/peopleops/compensation.md

```diff
@@ -73,4 +71,6 @@ awarded to be expensed for a dinner out. The bonus is awarded per release.
 
 ### Health Care
 
-FlowForge offers a health care package for all employees in the United States, Canada, and India.
+FlowForge offers a health care package for all employees in the United States, Canada, and India. 
```

Comment:
What coverage in the US would be great to add too. Vision and dental is included